automated_analysis.py

```diff
@@ -464,5 +459,42 @@ def set_survey_percentages(survey_counts, total_survey_counts):
         fig.update_xaxes(tickangle=-60)
         fig.write_image(f"{automated_analysis_output_dir}/graphs/{plan.raw_field}_by_gender_normalised.png",
                         scale=IMG_SCALE_FACTOR)
+    
+    # Export safe to share raw messages for each episode to share with WUSC
+    log.info("Exporting safe to share raw messages for each episode...")
+    safe_to_share_messages = []  # of dict
```

Comment:
Can you say what the type of the dicts is e.g. # of dict of ... -> ...

Reply:
updated thanks